src/ploneintranet/search/browser/searchresults.py

```diff
@@ -22,7 +22,7 @@ def search_response(self):
             if form.get(facet):
                 facets[facet] = form.get(facet)
 
-        search_util = getUtility(ISiteSearch, name='example')
+        search_util = getUtility(ISiteSearch, name='zcatalog')
         response = search_util.query(
```

Comment:
Agreed. There is the potential to add an additional abstraction layer here, but I think we should attack that later.